### ISSUE TITLE
multiple solutions

### ISSUE THREAD

**Issue by na3na3na** [domschrei/mallob#18]
Does mallob support finding multiple answers for a CNF problem? I tried using -s2f to write the solutions to a file but i am only getting one solution. Is there another option to toggle to get multiple solutions?

**Comment by domschrei** [comment]
No, there is no native support for this. Mallob is not a \#SAT solver / model enumerator and only outputs one (arbitrary) model.

You can however get multiple models with incremental solving (which I believe you are using?) by forbidding all previous models. You can do that by adding a clause of length $n$ that encodes the negated model. If you don't need too many different models I could imagine this being reasonably efficient.
